### DISCUSSION TITLE
The tokenId field

### DISCUSSION THREAD

**Discussion by szanuje** [Zimtente/ens-collections#228] [Ideas]
Hey, I was thinking about the optionality of tokenId field. If we make it must have, I see a big advantage for relational databases. You can store that as decimal, indexed field (even make that an ID) and create foreign key constraints to join tables (e.g. table with opensea listings) based on tokenId, instead of using names/labels strings. I think making this field required is good step towards efficient data processing/querying, as the clubs collections grows really fast.

**Comment by tunnckoCore** [comment]
That's exactly what I'll do on top of the basic standard -- some relations.

But it's out of the scope for here.

What you mean by required? The upcoming format will always have `label: tokenId` in the json. It won't be required when people are giving just names (or files), I already handled that.

**Reply by szanuje** [reply]
@tunnckoCore when should we expect new format being introduced?

**Reply by tunnckoCore** [reply]
@szanuje in the coming days. I'm ready with everything, just updating my tooling packages.

Also, maybe @Zimtente was going to move the repo to my account? Either way, I'll start from another clean branch and replace everything.
